src/main/java/de/unistuttgart/iste/gits/quiz_service/controller/QuizController.java

```diff
@@ -78,4 +79,9 @@ public Quiz setNumberOfRandomlySelectedQuestions(@Argument int numberOfRandomlyS
         return quizService.setNumberOfRandomlySelectedQuestions(quizMutation.getAssessmentId(), numberOfRandomlySelectedQuestions);
     }
 
+    @MutationMapping
+    public Quiz logQuizCompleted(@Argument QuizCompletedInput input, @ContextValue LoggedInUser currentUser) {
```

Comment:
doesn't that need the context configuration of lukas?

Reply:
no you have add that configuration class like in the other services

Reply:
done